src/transport/Session.h

```diff
@@ -244,6 +244,12 @@ class Session
     virtual bool AllowsLargePayload() const                              = 0;
     virtual const SessionParameters & GetRemoteSessionParameters() const = 0;
     virtual System::Clock::Timestamp GetMRPBaseTimeout() const           = 0;
+
+    // Returns true if `subjectDescriptor.IsCommissioning` (based on Core Specification
+    // pseudocode in ACL Architecture chapter) should be true when computing a
+    // subject descriptor for that session.
+    virtual bool IsCommissioningSession() const { return false; }
```

Comment:
Should document that this is only valid to call during synchronous processing of a message received on the session.

Reply:
done